lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -2553,6 +2553,10 @@ IRBuilderAsmJs::BuildInt2(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::RegSlot
         }
 
         break;
+    
+    case Js::OpCodeAsmJs::Eqz_Int:
+        instr = IR::Instr::New(Js::OpCode::CmEqz_I4, dstOpnd, srcOpnd, m_func);
```

Comment:
I believe the optimizer will recognize this and change to a test reg, reg; jne (if used in an if condition). But we should validate. If it doesn't, better to update optimizer, which may have more general benefit, than add new op for this special case.